src/styles/testimonials.scss

```diff
@@ -0,0 +1,67 @@
+.testimonials {
+  @include gridOn1024px;
+
+  &__content {
+    grid-column: 6 / 11;
+  }
+
+  &__title {
+    grid-column: 1 / 6 ;
```

Comment:
Better move all your 'grid' code to `grid.scss` file.

Detailed video  [heve](https://mate.academy/learn/css-advanced/css-advanced-landing#/video/1310)